src/pudl/dbt_wrapper.py

```diff
@@ -75,14 +122,27 @@ def build_with_context(
     dbt = dbtRunner()
     dbt_dir = PUDL_ROOT_PATH / "dbt"
 
-    # 2025-06-06 TODO: use the --project-dir flag instead of chdiring all the time?
     with chdir(dbt_dir):
-        _ = dbt.invoke(["deps"])
-        _ = dbt.invoke(["seed"])
+        dbt.invoke(["deps"])
```

Comment:
When running `run-operation` I realized there's no need to assign `_ = i_dont_care_about_the_output()`... some weird brain fart led to this in the first place.